vulcanai2/tests/models/test_cnn.py

```diff
@@ -81,6 +89,21 @@ def test_forward_class_not_nan(self, cnn_class):
         assert np.any(~np.isnan(class_output))
         assert np.any(~np.isnan(raw_output))
 
+    @pytest.mark.xfail # TODO: Throws RuntimeError: sizes must be non-negative
+    def test_forward_pass_input_net_not_nan(self, 
+                                cnn_class_add_input_network,
+                                cnn_noclass):
+        """Confirm out is non nan."""
+        test_input = torch.ones([1, *cnn_noclass.in_dim])
+        test_dataloader = DataLoader(TensorDataset(test_input, test_input))
+        raw_output = cnn_class_add_input_network.forward_pass(
+            data_loader=test_dataloader,
+            convert_to_class=False)
```

Comment:
@rfratila any idea why this is failing? getting `RuntimeError: sizes must be non-negative`

Reply:
I doubt this has something to do with casting? not sure

Reply:
see my comment about the fixture about this error